map.css

```diff
@@ -0,0 +1,100 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+body{
+    font-family: Inter, sans-serif;
+    font-feature-settings: 'liga' 1, 'calt' 1; /* fix for Chrome */
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.flexcolumn-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    margin: 0 auto;
+}
+
+.title-container{
+    width: 40vw;
+    border: 3px solid black;
+    margin-top: 10vh;
+    border-radius: 5px;
+}
+.title-container h1{
+    font-size: 30px;
+    margin: 0;
+    padding: 5px;
+    background-color: #D9D9D9;
+
+}
+.title-container p{
+    padding: 10px;
+    font-size: 20px;
+    text-align: left;
+}
+
+.button-container{
+    width: 50vw;
+    margin-top: 10vh;
+    display: flex;
+    justify-content: start;
```

Comment:
```suggestion
    justify-content: center;
```

Reply:
This will fix Ki's second suggestion of "centering the regions".

From 
![Screen Shot 2024-03-27 at 11 35 57 AM](https://github.com/chingu-voyages/v48-tier1-team-05/assets/51532684/d4f90aa0-919b-4006-bd3f-28217573dd7a)

To
![Screen Shot 2024-03-27 at 11 36 21 AM](https://github.com/chingu-voyages/v48-tier1-team-05/assets/51532684/d4df2add-705b-4108-8e59-4cebb1452ffe)